shell/shell.go

```diff
@@ -44,16 +45,13 @@ func Run(ctx context.Context, command string, opts ...RunOption) error {
 		return err
 	}
 
-	r := &interp.Runner{
-		Dir: wd,
-		Env: env,
-
-		Exec: interp.DefaultExec,
-		Open: interp.OpenDevImpls(interp.DefaultOpen),
-
-		Stdin:  os.Stdin,
-		Stdout: os.Stdout,
-		Stderr: os.Stderr,
+	r, err := interp.New(func(r *interp.Runner) error {
+		r.Dir = wd
+		r.Env = env
+		return nil
+	})
```

Comment:
Note that what you're doing here isn't supported either - you should use `interp.Dir` and `interp.Env`. For example, doing it the current way has no guarantee that the `Dir` field will be an absolute path.

Reply:
i'm also realizing that i don't need to do this at all - our specified behavior here is exactly what the defaults do anyways. 

(which makes sense given this code was taken from a  pre-`interp.New` example..)